### PR TITLE
added a markdown parser to directly render the markdownfiles

### PR DESCRIPTION
**Pull request by TheEbolaDoc**
Signed-off-by: Christian Heusel <christian@heusel.eu>

### CODE REVIEW

File: muesli/__init__.py [hunk 1]
```diff
@@ -33,6 +34,16 @@
 databaseName = config['database']['connection']
 PRODUCTION_INSTANCE = config.get("production", True)
 
+# Read in the dataprotection and changelog so they are static to the instance
+dataprotection_path = os.path.join(os.path.dirname(__file__), "web", "static", "datenschutzerklaerung.md")
+with open(dataprotection_path) as f:
+    dataprotection = f.read()
+DATAPROTECTION_HTML = markdown(dataprotection)
```
Comment (tynsh) [inline muesli/__init__.py]: Please make this a function. If I understand that correctly, both, the gdpr and changelog page, are rendered on each request. If you make this a function, it is still easy to get the html string by calling like, dataprotection() and it is not rendered on each request.

File: muesli/web/views.py [hunk 2]
```diff
@@ -177,19 +177,7 @@ def emailUsers(request):
...
-        for line in part.split('\n')[1:]:
-            if line.startswith('topic:'):
-                pass
-            elif line.startswith('concerns:'):
-                pass
-            else: text.append(line)
-        text = '\n'.join(text)
-        entries.append({'date': date, 'description': text})
-    return {'entries': entries}
+    return {"html": CHANGELOG_HTML}
```
[the remark this answers is not in the document]
Comment (tynsh) [reply muesli/web/views.py]: why not render the markdown here? This seems like a better solution.

[the remark this answers is not in the document]
Comment (TheEbolaDoc) [reply muesli/web/views.py]: Rendering the markdown directly in the function would mean that it is rendered every time the function is called.
The way I implemented it the rendering is done while the application starts.
This is based on the assumption that the changelog stays the same on one deployment.

File: muesli/__init__.py (hunk 1 above)
Comment (TheEbolaDoc) [reply muesli/__init__.py]: As I already said in my other comment I implemented it this way so it is not rendered every time.
I will test if the implementation behaves like I think

Comment (TheEbolaDoc) [reply muesli/__init__.py]: The `__init__.py` is only called upon initialization as I expected.

**Pull request merged**